src/Symfony/Contracts/Deprecation/README.md

```diff
@@ -0,0 +1,22 @@
+Symfony Deprecation Contracts
+=============================
+
+A generic function and convention to trigger deprecation notices.
+
+This package provides a single global function named `deprecated()`.
+Its purpose is to trigger deprecations in a way that can be silenced on production environments
+by using the `zend.assertions` ini setting and that can be caught during development to generate reports.
+
+The function requires at least 3 arguments:
+ - the name of the package that is triggering the deprecation
```

Comment:
Should probably mention that we are talking about the Composer package name here to avoid any ambiguity.

Reply:
updated to "the name of the *Composer* package [...]"